modules/container-insights/main.tf

```diff
@@ -1,8 +1,14 @@
 ## kubernetes container-insights
 
+locals {
+  metrics_enabled = lookup(var.features, "enable_metrics", false)
+  logs_enabled    = lookup(var.features, "enable_logs", false)
+}
```

Comment:
Changing the enabled var from a boolean to a map is a breaking change that should come with a major version bump.
But I am ok with that because I think nobody is using enabled variable right now.